backend-modules/koios/src/main/java/com/bloxbean/cardano/client/backend/koios/KoiosEpochService.java

```diff
@@ -5,27 +5,75 @@
 import com.bloxbean.cardano.client.api.model.Result;
 import com.bloxbean.cardano.client.backend.api.EpochService;
 import com.bloxbean.cardano.client.backend.model.EpochContent;
+import com.bloxbean.cardano.client.plutus.util.PlutusOps;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
 import rest.koios.client.backend.api.epoch.model.EpochInfo;
 import rest.koios.client.backend.api.epoch.model.EpochParams;
 
+import java.io.IOException;
 import java.math.BigDecimal;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.util.*;
+import java.util.concurrent.atomic.AtomicInteger;
 
 /**
  * Koios Epoch Service
  */
 public class KoiosEpochService implements EpochService {
 
+    private final ObjectMapper objectMapper = new ObjectMapper();
+    private final static String PLUTUS_COST_MODELS_URL = "https://raw.githubusercontent.com/input-output-hk/plutus/master/plutus-core/cost-model/data/builtinCostModel.json";
```

Comment:
@edridudi Do we really need this dependency? As discussed earlier, it can just rely on operations defined in the PlutusOps class, and we will update this class whenever there is a new Plutus version. 

I don't think we should depend on a file from an external repository, as this code will break if the file is moved to a different location.

Reply:
it's there by mistake I will remove